server/odc-service/src/main/java/com/oceanbase/odc/service/git/model/GitRepositoryMapper.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.oceanbase.odc.service.git.model;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import com.oceanbase.odc.common.crypto.TextEncryptor;
+import com.oceanbase.odc.core.shared.PreConditions;
+import com.oceanbase.odc.metadb.git.GitRepositoryEntity;
+import com.oceanbase.odc.service.encryption.EncryptionFacade;
+
+import lombok.NonNull;
+
+/**
+ * @author: liuyizhuo.lyz
+ * @date: 2024/7/29
+ */
+@Component
+public class GitRepositoryMapper {
```

Comment:
`org.mapstruct.Mapper`
`org.mapstruct.Mapping`

will help do this~

Reply:
Thanks, I've replaced this with it.